r1cs/gadgets/std/src/fields/fp12.rs

```diff
@@ -820,11 +909,14 @@ where
 }
 
 impl<P, ConstraintF: PrimeField + SquareRootField> TwoBitLookupGadget<ConstraintF> for Fp12Gadget<P, ConstraintF>
-where
-    P: Fp12Parameters,
-    <P::Fp6Params as Fp6Parameters>::Fp2Params: Fp2Parameters<Fp = ConstraintF>,
+    where
+        P: Fp12Parameters,
+        <P::Fp6Params as Fp6Parameters>::Fp2Params: Fp2Parameters<Fp = ConstraintF>,
 {
     type TableConstant = Fp12<P>;
+    /* Two bit lookup circuit, chooses c[i], i=0,..,3, according to i = b[1]*2 + b[0] given
```

Comment:
Since that each implementation from Fp2Gadget above uses the underlying FpGadget implementation, there's no need to comment the same thing again. I would remove it here and in all the other files (except fp.rs)